src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
+    count++;
+
+    if (count <= 3) {
+      return (y) => x + y;
+    }
+
+    if (count > 3 && count % 2 === 0) {
+      return () => 'Bzzz... Error!';
+    } else {
+      return (y) => x + y; ;
+    }
```

Comment:
You have adder function duplicated twice, that's not okay. All you need is only one `if` block